gulpfile.js

```diff
@@ -39,7 +41,7 @@ gulp.task('styles', gulp.series(
  */
 gulp.task('copy:assets', () => {
   return gulp.src(`${configPaths.src}assets/**/*`)
-    .pipe(gulp.dest(`${destination}/assets/`))
+    .pipe(gulp.dest(slash(join(destination, 'assets'))))
```

Comment:
**question**: As we use `join`, won't Gulp and Node sort themselves out without us having to `slash` regarding Windows vs. Posix paths?

Reply:
Sorry @romaricpascal there's a note in the commit message:

>Note: Gulp globs use `slash()` to ensure Windows paths are flipped back to POSIX for compatibility

It's because [Gulp uses its own glob library](https://github.com/gulpjs/glob-parent#windows) and things (can) go wrong in strange ways. Sadly too `require('path/posix').normalize` doesn't flip Windows slashes either

Reply:
Ok, cool. I'm not sure if we'd use globs on `gulp.dest` but better safe than sorry.

Reply:
Yeah, definitely. Also affects non-glob things like `./` and `../` in joined paths too